browser/ntp_sponsored_images/view_counter_model.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_NTP_SPONSORED_IMAGES_VIEW_COUNTER_MODEL_H_
+#define BRAVE_BROWSER_NTP_SPONSORED_IMAGES_VIEW_COUNTER_MODEL_H_
+
+class ViewCounterModel {
+ public:
+  ViewCounterModel();
+  ~ViewCounterModel();
+
+  ViewCounterModel(const ViewCounterModel&) = delete;
```

Comment:
use `DISALLOW_COPY_AND_ASSIGN`

Reply:
OK to leave as-is, IMO. See note at https://github.com/brave/brave-core/pull/4075/files/2f7f51f8a0ffee135d823311f7bb464dd74e6acc#r370352211